app/views/account/contacts/destroy_error.js.erb

```diff
@@ -0,0 +1,2 @@
+// show error message to user
+$('.error_message').replaceWith($("<h4 style='color: red;'>Error occurred, please refresh the page!</h4>"));
```

Comment:
looks not so well.. you are replacing `.error_message` `DIV` with `H4`, also w/o `.error_message`
so in case if error appeared 2d time it won't be showed on a page

Reply:
Changed to 'append' method.